components/support/nimbus-fml/src/util/loaders.rs

```diff
@@ -435,7 +563,7 @@ impl FileLoader {
 
     fn lookup_repo_path(&self, user: &str, repo: &str) -> Option<&FilePath> {
         let key = format!("{}/{}", user, repo);
-        self.config.get(&key)
+        self.repo_refs.get(&key)
```

Comment:
Could you also change `config` into `repo_refs` in the comments (e.g. [here](https://github.com/mozilla/application-services/pull/6021/files#diff-bdf9218444ff173c7ff5b1d936efff35506f6d3464564f42ed5c189919f4d1c8R538)).